qaengine/cliengine.go

```diff
@@ -81,8 +81,14 @@ func (*CliEngine) fetchSelectAnswer(prob qatypes.Problem) (qatypes.Problem, erro
 		Options: prob.Options,
 		Default: def,
 	}
-	if err := survey.AskOne(prompt, &ans); err != nil {
-		logrus.Fatalf("Error while asking a question : %s", err)
+	if prob.Validation != nil {
```

Comment:
Dont' we want the validation in httprestengine, defaultengine and storeengines too?